src/pages/home/components/carousel/carousel.css.ts

```diff
@@ -0,0 +1,84 @@
+import { color } from "@/shared/styles/tokens/color.css";
+import { typographyVars } from "@/shared/styles/typography.css";
+import { style } from "@vanilla-extract/css";
+
+export const container = style({
+  position: "relative",
+  width: "100%",
+});
+
+export const header = style({
+  position: "absolute",
+  top: "0",
+  left: "0",
+  right: "0",
+  backgroundColor: color.black[200],
+  width: "100%",
+  padding: "1rem 1.6rem",
+  zIndex: "10",
+
+  display: "flex",
+  justifyContent: "center",
+});
+
+// 캐러셀 스타일
+export const carouselContainer = style({
+  width: "100%",
+  height: "100%",
+  overflow: "hidden",
+});
+
+export const carouselTrack = style({
+  display: "flex",
+  width: "100%",
+  height: "100%",
+  transition: "transform 0.5s ease-in",
+});
```

Comment:
```suggestion
    touchAction: "pan-y",
});
```

pay-y: 세로 방향으로 스크롤/드래그 허용, 좌우 스와이프는 차단